rwengine/src/engine/SaveGame.cpp

```diff
@@ -539,7 +539,11 @@ bool SaveGame::loadGame(GameState& state, const std::string& file)
 	auto icv = iconv_open("UTF-8", "UTF-16");
 	char* saveName = (char*)state.basic.saveName;
 
+#if defined(RW_NETBSD)
+	iconv(icv, (const char**)&saveName, &bytes, &outCur, &outSize);
```

Comment:
```
     size_t
     iconv(iconv_t cd, const char ** restrict src, size_t * restrict srcleft,
         char ** restrict dst, size_t * restrict dstleft);
```

POSIX shifted to const-less version and other OS-es (except NetBSD, ...) keep using const.

Is ther needed `const_cast<>`?

It depends on warning level - if I remember correctly (I patched it 2 days ago) it errored.